.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "3.8"
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+# command to run tests
+script:
+  - nosetests
```

Comment:
Whoops! Nice solve here @Chulstro with realizing that we needed the testing script here.

Reply:
Now I don't have to keep looking this up!